builtin/logical/pki/test_helpers.go

```diff
@@ -436,3 +436,27 @@ func performOcspPost(t *testing.T, cert *x509.Certificate, issuerCert *x509.Cert
 	require.NoError(t, err, "parsing ocsp get response")
 	return ocspResp
 }
+
+func requireCertMissingFromStorage(t *testing.T, client *api.Client, cert *x509.Certificate) {
```

Comment:
Are these actually used or is this for a future PR?

Reply:
They are being used within the corresponding ENT PR which mainly validate certificates are being stored within the appropriate PR and main clusters.